contracts/staking/src/msg.rs

```diff
@@ -194,7 +199,7 @@ pub enum QueryMsg {
 
 #[cw_serde]
 pub enum MigrateMsg {
-    V0_4_18ToV0_4_19 {},
+    V0_4_18ToV0_4_20 { send_fees_to_treasury: bool },
```

Comment:
Shouldn't this be `V0_4_19ToV0_4_20`?

Reply:
The version `0.4.19` has never been deployed so we are going to migrate from `0.4.18` to `0.4.20`